docs/import-data/csv/load-csv-clause.md

```diff
@@ -63,8 +63,8 @@ files into the container.
 
 Below, you can find two examples of how to use the LOAD CSV clause depending on
 the complexity of your data:
-- [Transferring CSV files into a Docker container {#transfer}](#transferring-csv-files-into-a-docker-container-transfer)
-- [Examples](#examples)
+- [Transferring CSV files into a Docker container](#transfer)
+- Examples
   - [One type of nodes and relationships](#one-type-of-nodes-and-relationships)
   - [Multiple types of nodes and relationships](#multiple-types-of-nodes-and-relationships)
   relationships](#multiple-types-of-nodes-and-relationships)
```

Comment:
This one is broken too, otherwise, all good 😄 

```suggestion
  - [Multiple types of nodes and relationships](#multiple-types-of-nodes-and-relationships)
```

Reply:
Fixed.